common/app/conf/switches/Switches.scala

```diff
@@ -46,6 +46,9 @@ trait SwitchTrait extends Switchable with Initializable[SwitchTrait] {
 
   def isSwitchedOn: Boolean = delegate.isSwitchedOn && new LocalDate().isBefore(sellByDate)
 
+  // true if switched on and switchboard has been read
```

Comment:
Change to "If the switchboard hasn't been properly initialised, then the "safe state" is returned, not the real switch value. This makes sure the switchboard is initialised before returning the answer"